File: gin/service/logs.go
package service

import (
	"webmis/config"
	"webmis/library"
)

/* 日志 */
type Logs struct{}

/* 写入数据库 */
func (Logs) File(file string, content string) {
	(&library.FileEo{}).New(config.Env().RootDir)
	(&library.FileEo{}).WriterEnd(file, content)
}

File: java/src/main/java/webmis/service/Logs.java
package webmis.service;

import webmis.config.Env;
import webmis.library.FileEo;

/* 日志 */
public class Logs extends Base {

  /* 写入文件 */
  public static void File(String file, String content) {
    FileEo.Root = Env.root_dir;
    FileEo.WriterEnd(file, content+"\n");
  }
  
}
